odl/tomo/geometry/geometry.py

```diff
@@ -85,6 +98,11 @@ def __init__(self, ndim, motion_part, detector, translation=None):
         # Cache geometry-related objects for backends that require computation
         self.__implementation_cache = {}
 
+        # Make sure there are no leftover kwargs
```

Comment:
This really should be added to more places, i've had errors N times because of stuff like this where I think i gave an argument but it was simply ignored.